graph/Guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Microsoft Graph REST API Guidelines
+
+## History
+
+| Date        | Version | Notes                                               |
+| ----------- | ------- | --------------------------------------------------- |
+| 2021-Mar-31 | v3.3    | 1st public release of the Graph REST API Guidelines |
+
+## Introduction
+
+The Microsoft Graph REST API guidelines are an extension of the [Microsoft REST API guidelines][1]. Readers of this document are assumed to be also reading the [Microsoft REST API guidelines][1] and be familiar with them. Microsoft Graph guidance is a superset of the Microsoft API guidelines and services should follow them *except* where this document outlines specific differences or exceptions to those guidelines. This document does contain additional Microsoft Graph specific guidance and additional details.
+
+## Dictionary types
```

Comment:
suggestion from Dan, add a table of content

Reply:
we should probably duplicate the table of content from the main one and remove the ones that don't apply